statistics/histogram.go

```diff
@@ -386,6 +412,32 @@ func (hg *Histogram) equalRowCount(value types.Datum) float64 {
 	return 0
 }
 
+// equalRowCountV2 estimates the row count where the column equals to value.
+func (hg *Histogram) equalRowCountV2(value types.Datum) float64 {
```

Comment:
It seems `equalRowCountV2` is completely compatible with `equalRowCount`, so how about merging them into one function? Otherwise, other developers don't know which function they should use.